src/Razor/src/Microsoft.AspNetCore.Razor.VSCode.Extension/package.json

```diff
@@ -1,257 +1,269 @@
+          "when": "resourceLangId == aspnetcorerazor"
+        },
+        {
+          "command": "extension.showRazorHtmlWindow",
+          "when": "resourceLangId == aspnetcorerazor"
+        },
+        {
+          "command": "razor.reportIssue",
+          "when": "resourceLangId == aspnetcorerazor"
+        },
+        {
+          "command": "extension.configureRazorDevMode"
+        },
+        {
+          "command": "extension.resetRazorDevModeConfiguration"
+        },
+        {
+          "command": "extension.razorActivated"
+        }
+      ]
     },
-    "scripts": {
-        "vscode:prepublish": "yarn run compile",
-        "clean": "rimraf dist",
-        "build": "yarn run clean && yarn run lint && tsc -p ./ && yarn run compile:TextMate",
-        "lint": "tslint --project ./",
-        "watch": "yarn run clean && yarn run lint && tsc -watch -p ./",
-        "compile:TextMate": "npx js-yaml syntaxes/aspnetcorerazor.tmLanguage.yml > syntaxes/aspnetcorerazor.tmLanguage.json"
-    },
-    "devDependencies": {
-        "@types/node": "9.4.7",
-        "@types/vscode": "1.31.0",
-        "cross-env": "^5.2.0",
-        "js-yaml": ">=3.13.1",
-        "rimraf": "2.6.3",
-        "ts-node": "^7.0.1",
-        "tslint": "^5.11.0",
-        "typescript": "3.3.4000"
-    },
-    "dependencies": {
-        "diff": ">=3.5.0",
-        "microsoft.aspnetcore.razor.vscode": "link:../Microsoft.AspNetCore.Razor.VSCode"
+    "configuration": {
+      "title": "Razor Configuration",
+      "properties": {
+        "razor.plugin.path": {
+          "type": [
+            "string",
+            "null"
+          ],
+          "default": null,
+          "description": "Overrides the path to the Razor plugin dll."
+        },
+        "razor.languageServer.debug": {
+          "type": "boolean",
+          "default": false,
+          "description": "Specifies whether to wait for debug attach when launching the language server."
+        },
+        "razor.format.enable": {
+          "type": "boolean",
+          "scope": "window",
+          "default": true,
+          "description": "Enable/disable default Razor formatter."
+        }
+      }
     }
+  },
+  "scripts": {
+    "vscode:prepublish": "yarn run compile",
+    "clean": "rimraf dist",
+    "build": "yarn run clean && yarn run lint && tsc -p ./ && yarn run compile:TextMate",
+    "lint": "eslint --ext .ts ./src",
+    "watch": "yarn run clean && yarn run lint && tsc -watch -p ./",
+    "compile:TextMate": "npx js-yaml syntaxes/aspnetcorerazor.tmLanguage.yml > syntaxes/aspnetcorerazor.tmLanguage.json"
+  },
+  "devDependencies": {
+    "@types/node": "9.4.7",
+    "@types/vscode": "1.31.0",
+    "cross-env": "^5.2.0",
+    "eslint": "6.8.0",
+    "js-yaml": ">=3.13.1",
+    "rimraf": "2.6.3",
+    "ts-node": "^7.0.1",
+    "typescript": "3.8.3"
+  },
+  "dependencies": {
+    "diff": ">=3.5.0",
+    "microsoft.aspnetcore.razor.vscode": "^0.0.1"
```

Comment:
Does changing this allow us to still debug into razor.vscode pieces when running locally?

Reply:
Yeah debugging still works.